src/dashboard/src/main/models.py

```diff
@@ -409,6 +409,10 @@ def __unicode__(self):
     def update_active_agent(self, user_id):
         UnitVariable.objects.update_active_agent("SIP", self.uuid, user_id)
 
+    def add_custom_identifier(self, scheme, value):
+        """Allow callers to add custom identifiers to the model's instance."""
+        self.identifiers.add(Identifier.objects.create(type=scheme, value=value))
```

Comment:
`self.identifiers.create`? I think this does the same thing, so it's not a big deal, it's just shorter the other way. If my suggestion isn't working please let me know and I'll stop suggesting it 😄

Reply:
Ahhh! Okay, I was just being slow! Unit tests and a quick play-through show that this works a treat! I'm surprised it worked the way I put it together at all! :smile: